lib/cuda_color_spinor_field.cu

```diff
@@ -1863,23 +1875,23 @@ namespace quda {
     return out;
   }
 
-//! for deflated solvers:
-  cudaColorSpinorField& cudaColorSpinorField::Eigenvec(const int idx) const {
+//! for composite fields:
+  cudaColorSpinorField& cudaColorSpinorField::Component(const int idx) const {
```

Comment:
Any particular reason why `Component` is only around for `cudaColorSpinorField` and not `ColorSpinorField` ? Apart from the obvious only implement what is currently needed?